client/src/modules/Job/components/titlePage/TitlePage.tsx

```diff
@@ -0,0 +1,15 @@
+import { Space, Typography } from "antd"
+import BadgeItem from "./BadgeItem"
+
+
+function TitlePage() {
```

Comment:
try to follow by one style of code everywhere. change declaration of function as arrow function.
```suggestion
const TitlePage = ()=> {
```